src/python/py/models/quantized_model.py

```diff
@@ -729,16 +841,16 @@ def __init__(self, module):
 
 class QuantModel:
     @staticmethod
-    def from_pretrained(quant_type, input_path, bits, group_size, use_g_idx, q_size, kv_size, intermediate_size, num_layers):
+    def from_pretrained(quant_type, input_path, bits, group_size, use_g_idx, q_size, kv_size, intermediate_size, num_layers, adapter_path=None):
```

Comment:
`None` can be removed since you already pass `None` in for when `adapter_path` isn't used.

```suggestion
    def from_pretrained(quant_type, input_path, bits, group_size, use_g_idx, q_size, kv_size, intermediate_size, num_layers, adapter_path):
```